### PR TITLE
Revamp the config system to use the JSON manifests

### PR DESCRIPTION
**Pull request by wkm97**
issue: https://github.com/nation3/app/issues/30

### CODE REVIEW

**Review by xPi2 — APPROVED** [review]
LGTM, not tested

**Comment by aahna-ashina** [comment]
@wkm97 Will this code change require any environment variable adjustments in Vercel?

File: ui/.env.goerli [hunk 1]
```diff
@@ -6,8 +6,7 @@ NEXT_PUBLIC_BALANCER_NATION_ETH_POOL_ID="0x0bf37157d30dfe6f56757dcadff01aed83b08
 NEXT_PUBLIC_BALANCER_NATION_ETH_POOL_TOKEN="0x161D5108C721C85F1Af4D4f6f790AaBAa37268e7"
```
Comment (aahna-ashina) [inline ui/.env.goerli:6]: @wkm97 If we change to the file in `contracts/deployments/goerli.json`, can this `ui/.env.goerli` file be deleted?

File: ui/lib/config.ts [hunk 2]
```diff
@@ -1,45 +1,46 @@
...
+  nationDropAmount: number,
 }
 
-if (process.env.NEXT_PUBLIC_CHAIN !== 'mainnet') {
-  const zeroAddress = '0x0000000000000000000000000000000000000000'
-  const devDeployments = require(`../../contracts/deployments/${process.env.NEXT_PUBLIC_CHAIN}.json`)
-  config = {
-    nationToken: devDeployments.nationToken || zeroAddress,
-    veNationToken: devDeployments.veNationToken || zeroAddress,
-    // @ts-expect-error ts-migrate(2322) FIXME: Type 'number' is not assignable to type 'string | ... Remove this comment to see the full error message
```
Comment (aahna-ashina) [inline ui/lib/config.ts:26]: @wkm97 Looks look you also deleted this TypeScript line? Was that on purpose?

**Review by aahna-ashina — CHANGES_REQUESTED** [review]
@wkm97 I'm getting an error message when launching your branch on http://localhost:42069/

I'm not sure if this error has anything do do with your code changes, though.

Are you experiencing the same issue?

Screen Shot 2022-06-27 at 3 50 39 PM

**Comment by xPi2** [comment]
@aahna-ashina I think I fixed, let me know if you have any problem now.
I also cleaned the .env files and merged with main.

**Review by luisivan — APPROVED** [review]
Just tried the preview build on Vercel and seems to be working!

**Comment by aahna-ashina** [comment]
@wkm97 Looks like your branch has merge conflicts that need to be resolved?

**Review by aahna-ashina — COMMENTED** [review]
@xPi2 I'm no longer getting the warning at the top of the page asking me to switch the Ethereum network.  Was that removed? 

Screen Shot 2022-06-30 at 8 22 24 PM

I remember earlier that there used to be a message at the top prompting the user to change the network?

Here is how I tested:
```
cd ui
yarn install

cp .env.goerli .env.local
yarn dev
```
Then 
I opened http://localhost:42069/ while connected to the Ethereum mainnet.

**Review by aahna-ashina — APPROVED** [review]

**Pull request merged**